README.md

```diff
@@ -179,6 +177,23 @@ class User {
 
 Queries that transform the encrypted column wont work because transformers and subscribers operate outside of the DBMS.
 
+### What alogorithm should I use?
+
+Unless you need to maintain compatibility with an older system (or you know exactly what you're doing),
+you should use "aes-256-gcm" for the mode.
+This means that the encryption keys are are 256 bits (32-bytes) long and that the mode of operation
+is GCM ([Galois Counter Mode](https://en.wikipedia.org/wiki/Galois/Counter_Mode)).
+
+GCM provides both secrecy and authenticity and can generally use CPU acceleration where available.
+
+### Should I hardcode the IV?
+
+No. Don't ever do this.
+It will break the encryption and is vulnerable to a "repeated nonce" attack.
+
+If you don't provide an IV, the library will randomly generate a secure one for you.
```

Comment:
Maybe "In production, hardcoding an IV is not secure. See repeated nonce attack." And link to https://www.elttam.com/blog/key-recovery-attacks-on-gcm/.